crates/swc/tests/fixture/issues-10xxx/11118/output/index.js

```diff
@@ -0,0 +1,7 @@
+function xxx() {
+    var b = [].concat(Array.prototype.slice.call(arguments));
```

Comment:
https://github.com/babel/babel/blob/main/packages/babel-plugin-transform-spread/src/index.ts

Babel optimizes away the `[].concat` here. We can iterate on a refinement later.